.gitignore

```diff
@@ -41,3 +41,8 @@ prof/
 # Settings
 .idea/
 **.code-workspace
+
+# Hide 'trick' files from file navigation
+/doc/tox.ini
+/expertsystem/solvers/constraint/tox.ini
+/tests/tox.ini
```

Comment:
Since these files are under version control, I don't understand why they have to be excluded

Reply:
Ah yes, this is actually from an older attempt. The correct way (at least for VSCode) is 48832b8f